mobile/lib/providers/album/album.provider.dart

```diff
@@ -94,3 +147,31 @@ final albumRenderlistProvider =
   }
   return const Stream.empty();
 });
+
+class LocalAlbumsNotifier extends StateNotifier<List<Album>> {
+  LocalAlbumsNotifier(this.db) : super([]) {
+    final query = db.albums.where().remoteIdIsNull();
+
+    query.findAll().then((value) {
+      if (mounted) {
+        state = value;
+      }
+    });
+
+    _streamSub = query.watch().listen((data) => state = data);
+  }
+
+  final Isar db;
```

Comment:
same as above: Isar is never used except in the constructor. yet this makes it externally available

Reply:
You need to define it in the class instance to use it in the constructor, right?

<img width="819" alt="image" src="https://github.com/user-attachments/assets/10482db1-43e4-4d51-b2d0-a0c1f6f6fa01">